src/features/ITGovernance/Admin/GRBReview/GRBReviewForm/Presentation/index.tsx

```diff
@@ -1,60 +1,509 @@
+          recordingPasscode: links.recordingPasscode,
+          transcriptLink: links.transcriptLink
+        }
+      }
+    });
+  };
 
   return (
-    <EasiFormProvider<PresentationFields> {...form}>
-      <GRBReviewFormStepWrapper<PresentationFields>
-        grbReview={grbReview}
-        // TODO: Update onSubmit to return mutation
-        onSubmit={(async () => null) as unknown as GRBReviewFormStepSubmit}
-      >
-        <Controller
-          name="grbDate"
-          control={control}
-          render={({ field: { ref, ...field }, fieldState: { error } }) => (
-            <FormGroup error={!!error}>
-              <Label
-                htmlFor={field.name}
-                className="text-normal"
-                requiredMarker
-              >
-                GRB Date
-              </Label>
-
-              <DatePicker
-                {...field}
-                id={field.name}
-                defaultValue={field.value}
-              />
-            </FormGroup>
-          )}
-        />
-      </GRBReviewFormStepWrapper>
-    </EasiFormProvider>
+    <>
+      {grbReview.grbReviewType === SystemIntakeGRBReviewType.STANDARD ? (
+        <EasiFormProvider<StandardPresentationFields> {...standardForm}>
+          <GRBReviewFormStepWrapper
+            grbReview={grbReview}
+            onSubmit={onSubmitStandard}
+          >
+            <Grid desktop={{ col: 6 }}>
+              <div className="border-top-1px border-base-lighter margin-top-3 easi-text-normal">
+                <h4 className="margin-top-1 margin-bottom-05">
+                  {t('presentationGRBReviewForm.heading')}
+                </h4>
+
+                <p className="margin-y-0 text-base">
+                  {t('presentationGRBReviewForm.description')}
+                </p>
+
+                <Alert type="info" slim className="margin-top-105">
+                  {t('presentationGRBReviewForm.alert')}
+                </Alert>
+
+                <Controller
+                  name="grbMeetingDate.grbDate"
+                  control={control}
+                  rules={{
+                    required: 'presentationGRBReviewForm.required'
+                  }}
+                  render={({ field: { ref, ...field } }) => (
+                    <FormGroup>
+                      <Label
+                        htmlFor={field.name}
+                        className="text-normal"
+                        requiredMarker
+                      >
+                        {t('presentationGRBReviewForm.meetingDateLabel')}
+                      </Label>
+
+                      <HelpText className="margin-top-1">
+                        {t('presentationGRBReviewForm.meetingDateDescription')}
+                      </HelpText>
+
+                      <ErrorMessage
+                        errors={errors}
+                        name="grbMeetingDate.grbDate"
+                        render={({ message }) =>
+                          isSubmitted && (
```

Comment:
I don't think the `isSubmitted` conditional is needed here. The form should only validate fields and show error messages if the form has been submitted, unless you specify a different `criteriaMode` in `useEasiForm` ([example here](https://github.com/CMS-Enterprise/easi-app/blob/40a7875a7a5d6d49bb042b230f11505e86e996f6/src/features/ITGovernance/Admin/GRBReview/GRBReviewerForm/AddReviewerFromEua.tsx#L68)).

You should be able to render the `ErrorMessage` like this:
```
<ErrorMessage
  errors={errors}
  name="grbMeetingDate.grbDate"
  as={<FieldErrorMsg />}
/>
```